### ISSUE TITLE
Polarity switching and MS level parameters on main screen

### ISSUE THREAD

**Issue by Raghavdata** [ElucidataInc/ElMaven#457]
As told by Daniel:

"Every time I want to analyze data I have to first specify polarity and MS1/2 before I can load the data while this is just a few clicks, I was wondering if it wouldn't make sense to have those two options on the main screen. The polarity filter for instance doesn't have a function right now if you have to preemptively select the polarity anyway"

**Comment by Raghavdata** [comment]
@danielbraas It does make sense to have those parameters on the main screen. Having said that we will have to analyse how frequently the polarity switching and MS filter parameters are being used. This is because we feel the main screen already has quite a few parameters on it and adding parameters which are not used frequently might not be the best design choice.

Then again, to the best of my knowledge, the two parameters mentioned by you are used often. 

@sahil21 What are your thoughts?

**Comment by sahil21** [comment]
@danielbrass @Raghavdata We should try avoiding lots of parameters on the Main window especially those which are not generic to every dataset/workflow. 

But in near future, we'll work on something like saving a template of parameters and load them back. This will help you pre-define parameters in a file and load them back with much less effort. Check the related issue here #425

**Comment by danielbraas** [comment]
@sahil21 I agree that the main window shouldn't have too many 'buttons' but right now, you have a drop-down button for polarity that doesn't really seem to have much of a function. Once you set the parameters for the files to be loaded, the polarity button won't be useful anymore. My suggestion is (unless I don't see whatever other function the button would have) to adjust the loading settings with a click on that polarity button. This way, you wouldn't have to add anything to the main window. Would that work?
